templates/user/openid_manage.mako

```diff
@@ -1,6 +1,29 @@
 <%inherit file="/base.mako"/>
-<%namespace file="login.mako" import="render_openid_form" />
 <%def name="body()">
     <h2>Associate more OpenIDs</h2>
     ${render_openid_form( redirect, True, openid_providers )}
-</%def>
\ No newline at end of file
+</%def>
+
+<%def name="render_openid_form( redirect, auto_associate, openid_providers )">
+    <div class="toolForm">
+        <div class="toolFormTitle">OpenID Login</div>
+        <form name="openid" id="openid" action="${h.url_for( controller='user', action='login' )}" method="post" target="_parent" >
+            <div class="form-row">
+                <label>OpenID URL:</label>
+                <input type="hidden" name="method" value="redirect"/>
+                <input type="text" name="openid_url" size="60" style="background-image:url('${h.url_for( '/static/images/openid-16x16.gif' )}' ); background-repeat: no-repeat; padding-right: 20px; background-position: 99% 50%;"/>
```

Comment:
Similar to my other comment, I think not many users know what is an OpenID URL, or where to find it. Also, there is not different URLs for an identity provider (it is usually only one). We would still be safe relying on the URLs we know; i.e., using the following drop-down where user can choose an OpenID. Therefore, I suggest removing this textbox and show the dropdown only.

Reply:
Done.